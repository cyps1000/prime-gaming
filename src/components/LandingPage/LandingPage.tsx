/**
 * Imports the component styles
 */
import { useStyles } from "./LandingPage.styles";

/**
 * Component Imports
 */
import Banner from "../Banner";

/**
 * Displays the component
 */
const LandingPage: React.FC = () => {
  /**
   * Gets the component styles
   */
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <Banner />
    </div>
  );
};

export default LandingPage;
