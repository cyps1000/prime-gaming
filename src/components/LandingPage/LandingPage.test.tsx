/**
 * @see https://testing-library.com/docs/react-testing-library/intro
 * @see https://www.robinwieruch.de/react-testing-library
 */
import { render } from "@testing-library/react";

/**
 * External Imports
 */
import pretty from "pretty";

/**
 * Imports component
 */
import LandingPage from "./LandingPage";

/**
 * Default test
 */
describe("LandingPage Rendering Tests", () => {
  it("Renders the component without errors", () => {
    render(<LandingPage />);
  });
});

/**
 * LandingPage snapshot test
 */
describe("LandingPage Snapshot Test", () => {
  it("Passes the snapshot test", () => {
    const { container } = render(<LandingPage />);

    expect(pretty(container.innerHTML)).toMatchSnapshot();
  });
});
