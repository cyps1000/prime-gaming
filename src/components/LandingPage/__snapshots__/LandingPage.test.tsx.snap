// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage Snapshot Test Passes the snapshot test 1`] = `
"<div class=\\"makeStyles-root-5\\">
  <div class=\\"makeStyles-root-6 makeStyles-root-8\\"></div>
</div>"
`;
